public/app/features/dashboard-scene/scene/layout-rows/RowItemRenderer.tsx

```diff
@@ -193,6 +207,7 @@ function getStyles(theme: GrafanaTheme2) {
       '> div:nth-child(2)': {
         marginLeft: theme.spacing(3),
         position: 'relative',
+        width: `calc(100% - ${theme.spacing(3)})`,
```

Comment:
@bfmatei wait with this, this is not needed

Reply:
Yup, removed